ruby-challenges.rb

```diff
@@ -21,7 +30,15 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# created my method.
+def no_vowels(string)
+    result = string.gsub(/[aeiou]/i, "")
```

Comment:
Great use of regex!  Ruby also has another built-in method that has similar behavior.